python-avd/pyavd/_eos_designs/schema/schema_fragments/underlay_ospf_authentication_schema.yml

```
@@ -0,0 +1,40 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  underlay_ospf_authentication:
+    documentation_options:
+      table: ospf-settings
+    type: dict
+    keys:
+      enabled:
+        type: bool
+        default: false
+      message_digest_keys:
+        type: list
+        items:
+          type: dict
+          keys:
+            id:
+              type: int
+              convert_types:
+              - str
+            hash_algorithm:
+              type: str
+              valid_values:
+              - md5
+              - sha1
+              - sha256
+              - sha384
+              - sha512
+              default: sha512
+            key:
+              type: str
+              description: |
+                Key password.
+                Only plaintext passwords are supported here as the password will need to be encrypted for every underlay interface.
```

Can we reword this to state that AVD is doing the job of encrypting the password? here it feels like `will need to be encrypted` implies extra work required.

Something like:
```suggestion
                Only plaintext passwords are supported here, the per inteerface hashed password is automatically created by `eos_designs` using the relevant keys.
```